Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -660,3 +639,38 @@ internal extension CFURLSessionEasyCode {
         throw self
     }
 }
+
+#if os(Android)
+extension URLSession {
+    /// See https://curl.haxx.se/docs/sslcerts.html
+    /// For SSL to work you need a "cacert.pem" to be accessible at the
+    /// by setting URLSession.sslCertificateAuthorityFile to a file URL.
+    /// Downloadable here: https://curl.haxx.se/docs/caextract.html
+    /// Alternatively you can bypass SSL peer verification altogether:
+    /// URLSession.sslUnsafeBypassPeerVerify = true (not advised)
```

Comment:
These doc comments should be affixed to the corresponding properties. Here, they are documenting a `fileprivate static var`, which is not useful in autogenerated documentation.